cosmos/keybase.go

```diff
@@ -81,10 +85,25 @@ func (kb *Keybase) Delete(name, passphrase string, skipPass bool) error {
 }
 
 // Sign is a lock protected version of keys.Sign
+// it also keeps the last private key used in memory for the time set in `keepPrivTime` for performance improvement.
 func (kb *Keybase) Sign(name, passphrase string, msg []byte) ([]byte, crypto.PubKey, error) {
 	kb.mx.Lock()
 	defer kb.mx.Unlock()
-	return kb.kb.Sign(name, passphrase, msg)
+	hash := sha256.Sum256([]byte(name + passphrase))
```

Comment:
We need to find different way for calc hash. Imagine you have two users:

- bob:pass
- bobp:ass

They both will have the same hash which can lead to unexpected behaviour.

I think adding join string will solve the problem

```suggestion
	hash := sha256.Sum256([]byte(name + ":" + passphrase))
```